src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/group_convolution_backprop_data.cpp

```diff
@@ -74,7 +74,7 @@ INSTANTIATE_TEST_SUITE_P(smoke_GroupConvBackpropData_1d_AutoPadValid, GroupConvB
 /* ============= _2d GroupConvolution ============= */
 const std::vector<std::vector<ov::Shape>> input_shapes_2d = {{{1, 16, 10, 10}},
                                                              {{1, 32, 10, 10}}};
-const std::vector<std::vector<size_t >> kernels_2d = {{1, 1}, {3, 3}};
+const std::vector<std::vector<size_t >> kernels_2d = {{3, 3}, {2, 2}};
```

Comment:
Shouldn't we bring back the parameters, just like it has been done for `convolution_backprop_data`?

Reply:
Applied. also have acc issue and skip test cases  of failed.